cv.tex

```diff
@@ -31,41 +31,49 @@
 
 \begin{aside} % In the aside, each new line forces a line break
 \section{contact}
-+5548991060132
+Berlin,Germany
++39 349 859 0291
 \href{mailto:tiagokatcipis@gmail.com}{tiagokatcipis@gmail.com}
 \href{https://github.com/katcipis}{GitHub}
 \href{http://www.linkedin.com/pub/tiago-katcipis/1b/273/8b0}{LinkedIn}
 \href{http://katcipis.github.io/}{Blog}
-\section{languages skill}
-English: advanced, Portuguese: advanced
-\section{programming languages}
-Go, C, C++, Python, Lua, Nash, Bash, Javascript
-\section{protocols}
-HTTP, SIP, RTP, AMQP
-\section{cloud}
-Kubernetes, AWS, Azure, Google Cloud, Docker, CoreOS, Terraform
-\section{automation}
-Ansible, Make
-\section{dev environment}
-Vagrant, Docker Compose
-\section{monitoring}
+\section{Skills}
+Software architecture and design.
+Critical thinking and problem decomposition.
+Clear communication, written and spoken.
+\section{Programming Languages}
+Go, Python, C, C++, Lua, Bash, Javascript
+\section{Protocols}
+HTTP, gRPC, SIP, RTP, AMQP
+\section{Automation and Infrastructure}
+Kubernetes, AWS, Azure, Google Cloud, Docker, Terraform,
+Ansible, Make, Vagrant, Docker Compose
+\section{Monitoring}
 Prometheus, Grafana, StatsD, Sysdig
+\section{Languages}
+English, Portuguese
 \end{aside}
 
 %----------------------------------------------------------------------------------------
-%	EDUCATION SECTION
+%	INTRO SECTION
 %----------------------------------------------------------------------------------------
 
-\section{education}
+\section{Introduction}
 
 \begin{entrylist}
 %------------------------------------------------
 %------------------------------------------------
 \entry
-{2006--2011}
-{Bachelor of Computer Science}
-{UFSC}
```

Comment:
You could mention this at the end maybe

Reply:
ah true, I forgot to add that on the Skills section, the idea is to mention it in a more brief way, thanks for the review :heart: